src/Sulu/Bundle/ContactBundle/Controller/AccountController.php

```diff
@@ -171,7 +191,7 @@ public function __construct(
      */
     public function getContactsAction($id, Request $request)
     {
-        if ('true' == $request->get('flat')) {
+        if ('true' === $request->get('flat')) {
```

Comment:
I'm not sure if we should do this, because now the check fails, if the value is `true` (not `"true"`) ... But the `RequestParametersTrait` has a `getBooleanRequestParameter()` method exactly for this use case